cmd/livepeer/livepeer.go

```diff
@@ -130,6 +130,8 @@ func main() {
 	if *version {
 		fmt.Println("Livepeer Node Version: " + core.LivepeerVersion)
 		fmt.Printf("Compiler version: %s %s\n", runtime.Compiler, runtime.Version())
```

Comment:
Maybe we should change the logging here, too.   "Compiler version" --> "GO runtime version"

Reply:
s/go/golang ?

Reply:
fixed